website/docs/dbt-cli/install/homebrew.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Homebrew"
+---
+
+dbt Labs maintains Homebrew formulae for the four oldest and most popular adapter plugins.
+
+Reasons to use Homebrew:
+- You use MacOS for local development
+- You use dbt with one of Postgres, Redshift, Snowflake, or BigQuery
+- You're not sure if you have system requirements (such as python3) already installed
```

Comment:
Is this an "all of the above", or "any of the above"? If all of the above are true, are there significant reasons to not use Homebrew (that fit into a sentence)?